### PR TITLE
fix: format quick bar should only display one at a time

### PR DESCRIPTION
**Pull request by lawvs**
https://user-images.githubusercontent.com/18554747/209855823-edc7c90a-3074-4b09-a104-17356d12624a.mov

### CODE REVIEW

File: packages/blocks/src/components/format-quick-bar/index.ts [hunk 1]
```diff
@@ -30,6 +32,10 @@ export const showFormatQuickBar = async ({
   container?: HTMLElement;
   abortController?: AbortController;
 }) => {
+  // Abort previous format quick bar
+  globalAbortController.abort();
+  globalAbortController = abortController;
```
Comment (himself65) [inline packages/blocks/src/components/format-quick-bar/index.ts:37]: maybe undefined?

Comment (lawvs) [reply packages/blocks/src/components/format-quick-bar/index.ts:37]: the `abortController` has default parameters, and there are TypeScript guards.
https://github.com/toeverything/blocksuite/blob/7a691c366fbfb923919ca592b59a813c73047cca/packages/blocks/src/components/format-quick-bar/index.ts#L23

Comment (himself65) [reply packages/blocks/src/components/format-quick-bar/index.ts:37]: oh i see

**Review by Himself65 — APPROVED** [review]

**Review by doodlewind — APPROVED** [review]
LGTM, thanks for the fix!

**Pull request merged**